src/routes/index.js

```diff
@@ -1,14 +1,42 @@
-import { Auth } from "../modules/auth/pages";
-import { Main } from "../modules/main/pages";
+import Auth from "../modules/auth/index";
+import Main from "../modules/main/index";
+import Account from "../modules/account/index";
+import Announcement from "../modules/announcement/index";
```

Comment:
Like we talked about, if you want to encapsulate all pages with searchers into one module (`annoucements`) then subpages like `/annoucements/list` or `annoucements/new` should be its child. The inside routing of a module should be in the module and exported for example as `annoucementsRoutes` from the module entry point. Later in this file, it should be `concat` or `spread` to the main app routing const.